From 310a0955e9a80f360b6fc6b9f5d5f203c6fe2138 Mon Sep 17 00:00:00 2001
From: Finley Xiao <finley.xiao@rock-chips.com>
Date: Mon, 19 Mar 2018 14:20:13 +0800
Subject: [PATCH] clk: rockchip: px30: Make usb480m critical

Gpu 480M is from usb480m and the source clocks should be always on
if change gpu frequency after power off pd.

Change-Id: I11b5b05381e1745919b7137a64e4d334786cf433
Signed-off-by: Finley Xiao <finley.xiao@rock-chips.com>
---
 drivers/clk/rockchip/clk-px30.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/clk/rockchip/clk-px30.c b/drivers/clk/rockchip/clk-px30.c
index 698ccccbc607..db3130e905e6 100644
--- a/drivers/clk/rockchip/clk-px30.c
+++ b/drivers/clk/rockchip/clk-px30.c
@@ -938,6 +938,7 @@ static const char *const px30_pmucru_critical_clocks[] __initconst = {
 	"pclk_pmu_pre",
 	"hclk_usb_niu",
 	"pll_npll",
+	"usb480m",
 };
 
 static void __iomem *px30_cru_base;
-- 
2.35.3

